internal/pkg/db/mongo/models/addressable.go

```diff
@@ -37,7 +43,7 @@ type Addressable struct {
 	Origin     int64         `bson:"origin"`
 }
 
-func (a Addressable) ToContract() contract.Addressable {
+func (a *Addressable) ToContract() contract.Addressable {
```

Comment:
I don't think the receiver needs to be a pointer here. No internal state is mutated. Small potatoes.